src/schema/sources.ts

```diff
@@ -596,6 +611,13 @@ const validateSquadData = ({
     throw new ValidationError('Invalid member posting role');
   }
 
+  if (
+    typeof memberInviteRole !== 'undefined' &&
+    !sourceRoleRankKeys.includes(memberInviteRole)
+  ) {
+    throw new ValidationError('Invalid member invite role');
+  }
```

Comment:
Optional: Any way we can combine these 2 for future ease?

Reply:
I think verbosity is ok for now so I will leave it. Let's see where this evolves and then think about abstraction then. I think in general we have to see how and with what we want to validate across all the API routes.